api/app/GraphQL/Validators/PublishPoolAdvertisementInputValidator.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\GraphQL\Validators;
+
+use Carbon\Carbon;
+use Database\Helpers\ApiEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class PublishPoolAdvertisementInputValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        $endOfDay = Carbon::now()->endOfDay();
+        return [
+            // Pool name and classification
+            'name.en' => [ 'required', 'string' ],
+            'name.fr' => [ 'required', 'string' ],
+            'classifications.sync' => [ 'required', 'array', 'min:1' ],
+            'classifications.sync.*' => [ 'required', 'uuid', 'exists:classifications,id' ],
+
+            // Closing date (Note: Form should return unzoned datetime.)
+            'expiryDate' => [ 'required', /*'date_format:Y-m-d H:i:s',*/ 'after:'.$endOfDay ], // TODO: Fix date_format validation
```

Comment:
I see your notes here but I can't figure out a value that it will take.  I've tried a few different formats but no go.

![image](https://user-images.githubusercontent.com/8978655/181261890-c50d4129-5dcb-4c08-a783-99ca44474d51.png)

Reply:
Ya same here 😬 I was trying stuff from [PHP DateTime](https://www.php.net/manual/en/class.datetime.php)

Reply:
progress!

![image](https://user-images.githubusercontent.com/40485260/181298147-93f0be8e-2993-4c3a-bf10-8b13c50b5f23.png)

Reply:
it is broken though lol, the date check fails in PoolPolicy as a result

Reply:
Got it!  I should have looked at the schema notes for the right format.  :roll_eyes: 
![image](https://user-images.githubusercontent.com/8978655/181611970-98dc0d8c-22e7-4865-a353-a055a5fa5bae.png)